model_compression_toolkit/core/common/similarity_analyzer.py

```diff
@@ -186,6 +198,7 @@ def compute_lp_norm(float_tensor: np.ndarray,
         norm: whether to normalize the error function result.
         norm_eps: epsilon value for error normalization stability.
         batch: Whether to run batch similarity analysis or not.
+        axis: Axis along which the operator has been computed.
```

Comment:
if you only added axis for the signiture to be the same, better write that it's not used.